README.md

```diff
@@ -438,141 +450,149 @@ See [variables.tf] and [examples/] for details and use-cases.
   }
   ```
 
-  A `module_timeouts` object accepts the following fields:
+  The object accepts the following attributes:
 
-  - **`google_sql_database_instance`**: _(Optional `map(string)`)_
+  - [**`google_sql_database_instance`**](#attr-google_sql_database_instance-1): *(Optional `map(string)`)*<a name="attr-google_sql_database_instance-1"></a>
 
     - **`create`**: _(Optional)_ 
-      
+    
       Used for sql database instance creation. 
-      
+    
       Default is `"30m"`.
-
+    
     - **`update`**: _(Optional)_ 
-      
+    
       Used for sql database instance manipulation. 
-      
+    
       Default is `"30m"`.
-
+    
     - **`delete`**: _(Optional)_ 
-      
+    
       Used for sql database instance deletion.  
-      
+    
       Default is `"30m"`.
 
-  - **`google_sql_database`**: _(Optional `map(string)`)_
+    Default is `"30m"`.
+
+  - [**`google_sql_database`**](#attr-google_sql_database-1): *(Optional `map(string)`)*<a name="attr-google_sql_database-1"></a>
 
     - **`create`**: _(Optional)_ 
-      
+    
       Used for sql database creation. 
-      
+    
       Default is `"15m"`.
-
+    
     - **`update`**: _(Optional)_ 
-      
+    
       Used for sql database manipulation. 
-      
+    
       Default is `"10m"`.
-
+    
     - **`delete`**: _(Optional)_ 
-      
+    
       Used for sql database deletion.  
-      
+    
       Default is `"10m"`.
 
-  - **`google_sql_ssl_cert`**: _(Optional `map(string)`)_
+    Default is `"15m"`.
+
+  - [**`google_sql_ssl_cert`**](#attr-google_sql_ssl_cert-1): *(Optional `map(string)`)*<a name="attr-google_sql_ssl_cert-1"></a>
 
     - **`create`**: _(Optional)_ 
-      
+    
       Used for sql user creation. 
-      
+    
       Default is `"10m"`.
-
+    
     - **`delete`**: _(Optional)_ 
-      
+    
       Used for sql user deletion.  
-      
+    
       Default is `"10m"`.
 
-  - **`google_sql_user`**: _(Optional `map(string)`)_
+    Default is `"10m"`.
+
+  - [**`google_sql_user`**](#attr-google_sql_user-1): *(Optional `map(string)`)*<a name="attr-google_sql_user-1"></a>
 
     - **`create`**: _(Optional)_ 
-      
+    
       Used for sql user creation. 
-      
+    
       Default is `"10m"`.
-
+    
     - **`update`**: _(Optional)_ 
-      
+    
       Used for sql user manipulation. 
-      
+    
       Default is `"10m"`.
-
+    
     - **`delete`**: _(Optional)_ 
-      
+    
       Used for sql user deletion.  
-      
+    
       Default is `"10m"`.
 
+    Default is `"10m"`.
+
 #### Extended Resource Configuration
 
-- **`sql_databases`**: _(Optional `list(sql_databases)`)_
+- [**`sql_databases`**](#var-sql_databases): *(Optional `list(sql_databases)`)*<a name="var-sql_databases"></a>
 
   List of sql databases.
 
-  Example
+  Example:
 
   ```hcl
   sql_databases = [{
     name = "db"
   }]
   ```
 
-  Each `sql_databases` object accepts the following fields:
+  The object accepts the following attributes:
```

Comment:
@thiesen this should be fixed in terradoc i assume... ? when it's a list it should state "each object"..